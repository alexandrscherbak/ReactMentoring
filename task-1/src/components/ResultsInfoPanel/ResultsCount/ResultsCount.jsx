import React from 'react';

const ResultsCount = ({count}) => (
	<div>
		<span>{count} movies found</span>

		{/* <style jsx>
			{`
				span {
					font-size: 12px;
				}
			`}
		</style> */}
	</div>
);

export default ResultsCount;
