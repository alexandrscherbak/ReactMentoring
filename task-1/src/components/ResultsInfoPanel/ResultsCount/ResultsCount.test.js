import React from 'react';
import ResultsCount from './ResultsCount.jsx';
import renderer from 'react-test-renderer';

const props = {
	count: 12,
};

describe('ResultsCount', () => {
	test('snapshot', () => {
		const component = renderer.create(
			<ResultsCount {...props}/>,
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
