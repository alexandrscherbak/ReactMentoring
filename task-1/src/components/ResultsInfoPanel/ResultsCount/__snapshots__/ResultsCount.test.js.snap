// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsCount snapshot 1`] = `
<div>
  <span>
    12
     movies found
  </span>
</div>
`;
